rider-core/src/test/java/com/github/database/rider/core/MsSQLDatabaseIt.java

```diff
@@ -64,7 +73,7 @@ public void shouldSeedDataSetDisablingConstraints() {
     @DBUnit(leakHunter = true, caseSensitiveTableNames = true, escapePattern = "\"?\"")
     public void shouldFindConnectionLeak() throws SQLException {
         exception.expect(LeakHunterException.class);
-        exception.expectMessage("Execution of method shouldFindConnectionLeak left 1 open connection(s).");
+        exception.expectMessage("Execution of method shouldFindConnectionLeak left 2 open connection(s).");
```

Comment:
We can keep as `1` open connection here, this test with leak hunter is failing eventually and need to have a deeper look why. So please just keep as it was before

Reply:
Done